.claude/CLAUDE.md

```
@@ -0,0 +1,176 @@
+
+# Database Operations
+yarn db:migrate                 # Apply migrations  
+yarn db:migrate:dev             # Auto apply migrations, add new migrations if necessary
+yarn db:generate                # Generate the Prisma client
+yarn db:studio                  # Open Prisma Studio
+yarn db:drop                    # Drop all tables and reset the database
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)